### DISCUSSION TITLE
Multiple Time Series

### DISCUSSION THREAD

**Discussion by riyansn** [teamalgoritma/community#411] [Q&A] [answered]
Bagaimana cara memprediksi next game score dengan data seperti di screenshot ? apakah ada referensi yang bisa saya gunakan? saya menggunakan python dan tensorflow

terima kasih sebelumnya

![image](https://user-images.githubusercontent.com/52378435/197807865-79a37ed2-b2c8-4dcb-bc72-ffbc2d92c034.png)

**Comment by kinantytasyao** [comment, marked as answer]
Dear Bapak @riyansn ,

Untuk memprediksi game_score yang merupakan variabel numerik dengan beberapa variabel prediktor pada data Bapak dapat menggunakan linear regression. Sementara itu jika menggunakan tensorflow untuk linear regression Bapak dapat melihat referensinya pada link berikut https://www.geeksforgeeks.org/linear-regression-using-tensorflow/

Demikian yang dapat kami sampaikan, semoga membantu.
